Index: codespell/Makefile
===================================================================
--- codespell.orig/Makefile
+++ codespell/Makefile
@@ -27,8 +27,8 @@ check-dictionaries:
 			exit 1; \
 		fi; \
 	done
-	@if command -v pytest > /dev/null; then \
-		pytest codespell_lib/tests/test_dictionary.py; \
+	@if command -v pytest-3 > /dev/null; then \
+		pytest-3 codespell_lib/tests/test_dictionary.py; \
 	else \
 		echo "Test dependencies not present, install using 'pip install -e \".[dev]\"'"; \
 		exit 1; \
